src/Microsoft.AspNet.Mvc.Core/ActionResults/ViewExecutor.cs

```diff
@@ -70,7 +72,14 @@ public static class ViewExecutor
             {
                 try
                 {
-                    var viewContext = new ViewContext(actionContext, view, viewData, tempData, writer);
+                    var viewContext = new ViewContext(
+                        actionContext,
+                        view,
+                        viewData,
+                        tempData,
+                        writer,
+                        htmlHelperOptions);
```

Comment:
@rynowak the properties we care about are already members of `ViewContext`.  Are you suggesting:
- adding individual constructor parameters for the 4 of them
- passing in just the `HtmlHelperOptions` as @kirthik has done in the current iteration
- requiring _all_ "outermost" (i.e. not for an inner view or similar) `ViewContext` creators to set the individual properties
- something else?